src/components/Reports/ReportsPageRanking/ReportsPageRanking.styles.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const PageRankingItem = styled.div`
+  border-top: 1px solid #ccc;
+  border-bottom: 1px solid #ccc;
+  background: #f2f2f2;
```

Comment:
Here you could replace colors as  we've talked yesterday. As we did here https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/shared/C3Chart/C3Chart.styles.js